src/content/releases/release-v0.2.0.mdx

```diff
@@ -0,0 +1,119 @@
+left the company? Their account will now be automatically suspended,
+streamlining off-boarding without the hassle.
+
+As an admin click on the company dropdown -> Company Admin -> Manage Team Members.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/ljimvRg.png" alt="People Management" className="rounded-lg border border-gray-300" />
+</div>
+
+## Manage who does what on your projects
+
+We've upgraded how Operately handles project team changes. Now it's easier to
+match people's roles to their real responsibilities.
+
+Say your lead developer Sarah steps up to manage a critical feature launch. In
+seconds, you can promote her from contributor to project champion. She gets the
+right access, and you keep moving. Or three software engineers join the project, 
+each responsible for a different part of the system. Now, you add all three at once, 
+assigning them specific responsibilities and keeping everyone in the loop.
+
+Why does this matter? Because in the real world, roles change. People step up.
+New folks join. You shouldn't need an IT degree to keep your project moving when 
+this happens. We built this because we've felt this pain ourselves. Unclear 
+responsibilities kill momentum. They breed confusion.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/HX85OsP.png" alt="People Management" className="rounded-lg border border-gray-300" />
+</div>
+
+## Security and Compliance with One-Click Access Oversight
+
+We just fixed a headache you might not even know you had. It's about who has
+access to your projects. As your company grows, keeping track of who can see 
+what becomes a mess. It's a security risk waiting to happen. And when the auditors
+come knocking? Good luck. So we made it simple. Dead simple.
+
+We built this because we've been there. Digging through settings,
+cross-checking spreadsheets, wondering if we missed someone. It's a waste of
+time and it's nerve-wracking. Not anymore.
+
+Go to a project -> Manage Team & Access -> on the bottom click show all.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/mlQ5bZP.png" alt="See all people with access to the project" className="rounded-lg border border-gray-300" />
+</div>
+
+## Your Inbox, Your Rules: New Subscription Controls
+
+You can also control who will receive notifications in any type of a message
+that you send across spaces, projects, and goals.. Stay updated on the
+discussions that matter to you and tune out the ones that don’t. Whether it's
+project updates, comments, or notifications, you can easily manage what shows
+up in your inbox with just a click.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/vayerOI.png" alt="See all people with access to the project" className="rounded-lg border border-gray-300" />
```

Comment:
Its not clear what this screenshot is showing. It should be a gif with someone subscribing or unsubscribing from the message thread.